web-admin/src/routes/-/auth/device/+page.svelte

```diff
@@ -36,7 +31,16 @@
       }
     ).then((response) => {
       if (response.ok) {
-        successMsg = "User code confirmed, this page can be closed now";
+        if (redirectURL !== "") {
+          // This msg creates a tight coupling b/w login flow and github access flow
+          // but is required for better user experience
+          successMsg = "User code confirmed, verifying github access...";
+          setTimeout(function () {
+            window.location.href = decodeURIComponent(redirectURL);
+          }, 2 * 1000);
```

Comment:
It should redirect immediately. The next page should provide the relevant context about CLI auth succeeding, in addition to requesting the next action if relevant.

Reply:
Users can directly land on the next page (github connect) as well so we may need to add some additional context while redirecting which will need handling on backend as well. I will try to add a generic message like `Authentication successful` on the next page.